spec/features/idv/hybrid_mobile/hybrid_mobile_spec.rb

```diff
@@ -42,6 +42,10 @@
       visit idv_hybrid_mobile_capture_complete_url
       expect(page).to have_current_path(idv_hybrid_mobile_document_capture_url)
 
+      # Confirm that jumping to LinkSent page does not cause errors
+      visit idv_link_sent_url
```

Comment:
Presumably this will eventually cause problems with back button stuff, eh?

Reply:
Back button stuff is going to be tricky, yes.